### ISSUE TITLE
I don't understand this README.

### ISSUE THREAD

**Issue by satoichiki** [markstory/lint-review#109]
It is not written how to change image of docker.
and...
docker description are scattered far and wide.

**Comment by adrianmoisey** [comment]
Have you read this section: https://github.com/markstory/lint-review#using-docker ?

**Comment by satoichiki** [comment]
I think it has to write build command before ```docker-compose up -d broker worker web```.
like this.
```
$ docker build -t IMAGE_NAME.
```

I did before up command.
But it doesn't work...

**Comment by adrianmoisey** [comment]
> I think it has to write build command before docker-compose up -d broker worker web.
> like this.

That's wrong, because the build is now happening on hub.docker.com (see https://hub.docker.com/r/markstory/lint-review/), so there is no need to build the image first.

**Comment by satoichiki** [comment]
I did three steps after cloning this repository.

1. 
```
docker build -t ichiki/lint-review .
```

2.
```
docker-compose up -d broker worker web
```

3.
```
docker-compose run web lintreview register hotpepper-gourmet perolin.web
```

and then, it doesn't work...
```
curl http://127.0.0.1:5000/ping
curl: (7) Failed to connect to 127.0.0.1 port 5000: Connection refused
```

**Comment by adrianmoisey** [comment]
You can change this URL to use Github Enterprise: https://github.com/markstory/lint-review/blob/master/settings.sample.py#L66

**Comment by satoichiki** [comment]
I changed it.
Do I have to change something in docker-composer.yml?

**Comment by adrianmoisey** [comment]
There is an environment file that docker-compose uses, you can set it there. See https://github.com/markstory/lint-review/blob/master/docker-compose.yml#L12

**Comment by satoichiki** [comment]
Ok.
But now, I have this problem...
```
docker-compose run web lintreview register user_name repository_name
INFO 2016-06-20 06:59:07,672 github3 session:60 Building a url from ('https://api.github.com', 'repos', user_name, repository_name)
```

I don't know why it creates form api.github.com though I set the Enterprise of GITHUB_URL.

**Comment by markstory** [comment]
You might want to check that the environment variable is being picked up [by this code](https://github.com/markstory/lint-review/blob/master/lintreview/github.py#L15-L16).

**Comment by satoichiki** [comment]
I misunderstood.
I wrote env like github_user_name in settings.sample.py.
But It has to write in lintreview-env.

```lintreview-env
LINTREVIEW_GUNICORN_BIND=127.0.0.1:5000
GITHUB_URL=MYURL
GITHUB_USERNAME=MYUSERNAME
GITHUB_OAUTH_TOKEN=MYACCESSTOKEN
```

I understood it.
However, I have another problem.
I could connect with webhook.

But I got error message...
like this:
![2016-06-21 14 56 29](https://cloud.githubusercontent.com/assets/7994534/16219314/5da6568e-37c0-11e6-839a-b591c888ff52.png)

**Comment by adrianmoisey** [comment]
Looks like the host is invalid. Take a look at it to be sure that it's a valid URL

**Comment by satoichiki** [comment]
![2016-06-21 15 46 23](https://cloud.githubusercontent.com/assets/7994534/16220351/55c46760-37c7-11e6-8f56-09be1d1d97a8.png)

I'm using 127.0.0.1:5000.
I didn't get response when I tried to request the url.
```
$ curl http://127.0.0.1:5000/ping
curl: (7) Failed to connect to 127.0.0.1 port 5000: Connection refused
```
It seems that web server doesn't work.

**Comment by adrianmoisey** [comment]
127.0.0.1:5000 won't work. You need to ensure that your lintreview is on a public facing IP address accessible from the outside world.

**Comment by satoichiki** [comment]
I see.
However, I couldn't get response when I request ping in docker-machine.
![2016-06-21 16 06 32](https://cloud.githubusercontent.com/assets/7994534/16220821/2d680378-37ca-11e6-941e-9a25fbea8175.png)
It was empty...
I want to get this [response](https://github.com/markstory/lint-review/blob/master/lintreview/web.py#L21)

**Comment by satoichiki** [comment]
I could boot up web server.
I tried ```$ curl http://localhost:5000/ping  ```, but I  got  404 for the respond.
<img width="537" alt="2016-06-21 19 06 49" src="https://cloud.githubusercontent.com/assets/7994534/16225792/5cd60ff6-37e3-11e6-92f5-5bf06d953faa.png">

Do you happen to know how to solve this problem?
